include/gunrock/framework/operators/filter/predicated.hxx

```diff
@@ -82,46 +83,43 @@ void execute_mgpu(graph_t& G,
   input_ends[num_gpus - 1] = input->end();
```

Comment:
A minor detail, I prefer we do not use a for-loop or create the arrays here, we can just use the calculated chunk_size, to address it directly in the call to thrust/for-loop below.